NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs

```diff
@@ -24,20 +24,43 @@ public Optional<GameObject> Spawn(Entity entity, Optional<GameObject> parent)
             }
 
             GameObject gameObject = Utils.SpawnFromPrefab(prefab, null);
+
             gameObject.transform.position = entity.Position;
-            gameObject.transform.localScale = entity.Scale;
+            gameObject.transform.localScale = entity.LocalScale;
+
+            if (parent.IsPresent() && parent.Get().name == "CellRoot(Clone)")
+            {
+                LargeWorldEntity ent = gameObject.GetComponent<LargeWorldEntity>();
+                ent.transform.SetParent(parent.Get().transform, true);
+                ent.OnAddToCell();
+
+                gameObject.transform.localPosition = entity.LocalPosition;
+                gameObject.transform.localRotation = entity.LocalRotation;
+            }
+            else if (gameObject.name == "CellRoot(Clone)")
```

Comment:
Should we move this logic into a `TechType.None` entity spawner?

Reply:
In the event some gameObjects are passed to the TechType.None entity spawner id like to avoid that...

Reply:
I meant for us to move it while keeping the `if` check